core/templates/pages/story-editor-page/editor-tab/story-editor.directive.ts

```diff
@@ -90,32 +92,34 @@ angular.module('oppia').directive('storyEditor', [
 
           var _initEditor = function() {
             $scope.story = StoryEditorStateService.getStory();
-            $scope.storyContents = $scope.story.getStoryContents();
-            $scope.disconnectedNodes = [];
-            $scope.linearNodesList = [];
-            $scope.nodes = [];
-            $scope.allowedBgColors = (
-              storyConstants.ALLOWED_THUMBNAIL_BG_COLORS.story);
-            if ($scope.storyContents &&
-                $scope.storyContents.getNodes().length > 0) {
-              $scope.nodes = $scope.storyContents.getNodes();
-              $scope.initialNodeId = $scope.storyContents.getInitialNodeId();
-              $scope.linearNodesList =
-                $scope.storyContents.getLinearNodesList();
+            if ($scope.story) {
```

Comment:
How does this change the behaviour of this directive? What will happen when the story is `null` here?

Reply:
Okay, what is the behaviour when this remains undefined? This can be even seconds on slow connections.

Reply:
No effect in UI, it was just throwing console error "can read properties over undefined (reading getStoryContents()).

Reply:
Okay.